data/mods/Magiclysm/worldgen/orc_village.json

```diff
@@ -0,0 +1,64 @@
+[
+  {
+    "type": "palette",
+    "id": "orc_palette",
+    "terrain": { "3": "t_door_makeshift_c", "1": "t_wall_rammed_earth", ".": "t_dirtfloor" }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "orc_village",
+    "object": {
+      "fill_ter": "t_dirt",
+      "rows": [
+        "11111111111111111111111111111111111111111111111",
+        "1.............................................1",
+        "1.............................................3",
+        "1.............................................1",
+        "1.............................................3",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................3",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "1.............................................1",
+        "11111111111111111111111111111111111111111111111"
+      ],
+      "place_monsters": [
+        { "monster": "GROUP_ORC", "x": [ 5, 5 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] }
```

Comment:
```suggestion
        "11111111111111111111111",
        "1.....................1",
        "1.....................3",
        "1.....................1",
        "1.....................3",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................3",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "1.....................1",
        "11111111111111111111111"
      ],
      "place_monsters": [
        { "monster": "GROUP_ORC", "x": [ 3, 5 ], "y": [ 4, 23 ], "density": 0.2, "repeat": [ 1, 3 ] },
        { "monster": "GROUP_ORC", "x": [ 3, 21 ], "y": [ 4, 23 ], "density": 0.2, "repeat": [ 1, 3 ] },
        { "monster": "GROUP_ORC", "x": [ 3, 21 ], "y": [ 4, 23 ], "density": 0.2, "repeat": [ 1, 3 ] },
        { "monster": "GROUP_ORC", "x": [ 3, 21 ], "y": [ 4, 23 ], "density": 0.2, "repeat": [ 1, 3 ] }
```
I think it may want 24x24 unless doing a nested mapgen with mapgensize specification.